### PR TITLE
SCRUM-2225: upgrade quarkus to 2.13.3

### PR DESCRIPTION
**Pull request by abecerra**
plus some documentation and small code improvements

### CODE REVIEW

File: src/main/java/org/alliancegenome/mati/entity/SubdomainEntity.java [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+import lombok.*;
+import org.hibernate.Hibernate;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import java.util.Objects;
 
 @Entity(name = "Subdomain")
 @Table(name = "subdomain")
-@Data
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/mati/entity/SubdomainEntity.java:16]: Why the change from @Data? you can add the `@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)` annotation that will take care of this.

Comment (abecerra) [reply src/main/java/org/alliancegenome/mati/entity/SubdomainEntity.java:16]: I followed intelliJ suggestion and replacement: remove @Data for the entity to avoid problems in some cases

Comment (abecerra) [reply src/main/java/org/alliancegenome/mati/entity/SubdomainEntity.java:16]: Screenshot_20221028_101902
